glide.lock

```diff
@@ -42,7 +42,7 @@ imports:
 - name: github.com/lann/ps
   version: 62de8c46ede02a7675c4c79c84883eb164cb71e3
 - name: github.com/lib/pq
-  version: 8c6ee72f3e6bcb1542298dd5f76cb74af9742cec
+  version: 83612a56d3dd153a94a629cd64925371c9adad78
```

Comment:
All these packages updates are necessary?

Reply:
No, this was automatically updated by `glide up`.

Reply:
Then, just to avoid library incompatibility problems, can you only update the core-retrieval dependency?